### ISSUE TITLE
JSON-LD examples?

### ISSUE THREAD

**Issue by stain** [Nanopublication/Guidelines#19]
JSON-LD lends itself well to Nanopublications.  See for instance https://gist.github.com/stain/c2ad32ef4423b4814e83c2821bb03506 where I tried to reshape http://purl.org/np/RAW0X-iaUqouIVNNVYHBqbn5_LrVf2m3tv9l9K6B23tok as a nested JSON-LD structure that could also be consumed by non-RDF-aware consumers.

(BTW, that nanopub has somehow added `#head` also in the name of the nanopublication itself which I took the liberty to fix for the example)

**Comment by stain** [comment]
See also discussions about [Nanopubs on prov-comments](https://lists.w3.org/Archives/Public/public-prov-comments/2017Oct/)

**Comment by tkuhn** [comment]
Yes, JSON-LD examples would be nice. Did you notice that some nanopub servers actually give you JSON-LD, like here: http://np.inn.ac/RAW0X-iaUqouIVNNVYHBqbn5_LrVf2m3tv9l9K6B23tok.jsonld.txt

Sesame doesn't seem to do any prettification though.

I don't understand what you mean by the `#head` in the name of the nanopub. The given nanopub has just http://purl.org/np/RAW0X-iaUqouIVNNVYHBqbn5_LrVf2m3tv9l9K6B23tok as its identifier (without `#head`).

**Comment by stain** [comment]
Ah, yes, I see http://purl.org/np/RAW0X-iaUqouIVNNVYHBqbn5_LrVf2m3tv9l9K6B23tok correctly now. All praise TriG syntax for readability :-)  

Yes, I used that JSON-LD output as starting point; and we could use compaction against a common or generated JSON-LD `@context` - could that have its own Trusted URI to ensure it is unchanged -  or do we just include it explicitly as we do with the `@prefix`es?

We could in theory in the nanopub server use [JSON-LD Framing](https://json-ld.org/spec/latest/json-ld-framing/) as the latest draft [supports named graphs](https://json-ld.org/spec/latest/json-ld-framing/#framing-named-graphs), although I am not sure which implementations support this yet.
